### DISCUSSION TITLE
(generic) ESP32+SX1276 ?

### DISCUSSION THREAD

**Discussion by lebang1** [meshtastic/Discussions#43] [ESP32]
Hello, I have ESP32 DEVKIT V1 Board (Wi-Fi and Bluetooth) and RFM95w -868MHz.
Is there any guide about how to get it to work with meshtastic? Has Anybody made a software and connection guide?
![esp32-board-bg](https://github.com/user-attachments/assets/80770cdf-6248-4060-9e74-3b6e854f7977)

**Comment by egin10** [comment]
Hi there, have you try this one?
https://www.youtube.com/watch?v=zn1Tv9eQi5s

**Reply by lebang1** [reply]
Hello, thank you for that. I will try it out.
